staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -463,14 +463,13 @@ type PersistentVolumeClaimSpec struct {
 	// This is a beta feature.
 	// +optional
 	VolumeMode *PersistentVolumeMode `json:"volumeMode,omitempty" protobuf:"bytes,6,opt,name=volumeMode,casttype=PersistentVolumeMode"`
-	// This field requires the VolumeSnapshotDataSource alpha feature gate to be
-	// enabled and currently VolumeSnapshot is the only supported data source.
-	// If the provisioner can support VolumeSnapshot data source, it will create
+	// The Object Type in this field currently can be either a VolumeSnapshot, or an existing CSI PVC.
```

Comment:
@j-griffith, I think it should be in sync with this: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L422-L433

They should be the same and not mention `VolumePVCDataSource` feature gate.

In addition, `make update` will update all the generated APIs.

Reply:
@jsafrane Thanks... I thought that the changes in staging automatically propagated for some reason (or even vice versa) but apparently not, sorry.  So I guess update both?